### PR TITLE
Fix scale issues observed during jenkins run

### PR DESCRIPTION
**Pull request by ramkiperiy**
tests/e2e/scale/test_pvc_creation_deletion_scale.py
- Script enhanced to copy creation and deletion csv files to ocs-ci log location
- Script enhanced to parallel execution of code to collect pv and pvc names

tests/helpers.py
- Added waittime of 60 sec to measure_pv_creation_time_bulk
- Added waittime of 60 sec to measure_pv_deletion_time_bulk

Signed-off-by: Ramakrishnan <rperiyas@redhat.com>

### CODE REVIEW

**Comment by vasukulkarni** [comment]
why not add the ulimits via some function call here?

**Comment by ramkiperiy** [comment]
> why not add the ulimits via some function call here?

@vasukulkarni most of the time after changing this limit machine needs a reboot, which is tricky.

**Comment by vasukulkarni** [comment]
somehow the merge is disabled again, checking.

**Comment by ramkiperiy** [comment]
i don't see any issues while running tox in local laptop.
image

**Comment by petr-balogh** [comment]
> i don't see any issues while running tox in local laptop.
> image

Failed on docs - have you tried also `tox -e docs` ?

```
Warning, treated as error:
/home/travis/build/red-hat-storage/ocs-ci/ocs_ci/ocs/cluster.py:docstring of ocs_ci.ocs.cluster.CephCluster.cluster_name:1:duplicate object description of ocs_ci.ocs.cluster.CephCluster.cluster_name, other instance in apidoc/ocs_ci.ocs, use :noindex: for one of them
Makefile:20: recipe for target 'html' failed
make: *** [html] Error 2
ERROR: InvocationError for command /usr/bin/make html (exited with code 2)
```

Is this PR rebased on latest master?

I can take a look after holidays what's going on here but please make sure what I mentioned above, thanks.

**Comment by ramkiperiy** [comment]
> ```
> Warning, treated as error:
> /home/travis/build/red-hat-storage/ocs-ci/ocs_ci/ocs/cluster.py:docstring of ocs_ci.ocs.cluster.CephCluster.cluster_name:1:duplicate object description of ocs_ci.ocs.cluster.CephCluster.cluster_name, other instance in apidoc/ocs_ci.ocs, use :noindex: for one of them
> Makefile:20: recipe for target 'html' failed
> make: *** [html] Error 2
> ERROR: InvocationError for command /usr/bin/make html (exited with code 2)
> ```

Yeah observing this error when executing `tox -e docs` command, let me try to rebase the code and check.

**Comment by ramkiperiy** [comment]
@keesturam @vasukulkarni force pushed the code to resolve the tox issues, please approve this PR again, this need to be merged ASAP. Weekend Scale tests again failed, this PR will resolve the problem.

File: tests/helpers.py [hunk 1]
```diff
@@ -1086,20 +1086,23 @@ def measure_pvc_creation_time(interface, pvc_name):
...
+        wait_time (int): Seconds to wait before collecting CSI log
 
     Returns:
         pvc_dict (dict): Dictionary of pvc_name with creation time.
 
     """
     # Get the correct provisioner pod based on the interface
     pod_name = pod.get_csi_provisioner_pod(interface)
+    # due to some delay in CSI log generation added wait
+    time.sleep(wait_time)
```
Comment (RazTamir) [inline tests/helpers.py:1105]: Bug?

Comment (RazTamir) [reply tests/helpers.py:1105]: LGTM

**Pull request merged**